### ISSUE TITLE
Deprecation Warnings in Swift 4

### ISSUE THREAD

**Issue by rvoortman** [AlexanderNey/SemanticVersioning#17]
There are two deprecation warning in file `SemanticVersioningParser.swift` on line 232 and 247

```
/ios/Pods/SemanticVersioning/Source/SemanticVersioningParser.swift:232:79: 'characters' is deprecated: Please use String or Substring directly
```
It does not affect functionality, so I say it's a minor thing. Just wanted to let you know.

**Comment by igor9silva** [comment]
What tag are you pointing to? I must be fixed at `2.0.0`.

**Comment by rvoortman** [comment]
@igor9silva That solved the problem. Thank you for your reply!
